currency_scale.go

```diff
@@ -30,3 +32,14 @@ var (
 		"XPF": 0,
```

Comment:
Sorry for commenting late - but thinking about this - 
I think we are adding lot of currencies that we are probably not even using. Also adding comments to what is belonging to which country is also beneficial.